### ISSUE TITLE
Issue 1

### ISSUE THREAD

**Issue by GHcodingLady** [spyder-ide/spyder#6555]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "C:\Users\Lizzy\Anaconda3\lib\site-packages\spyder_profiler\widgets\profilergui.py", line 264, in <lambda>
    self.finished(ec, es))
  File "C:\Users\Lizzy\Anaconda3\lib\site-packages\spyder_profiler\widgets\profilergui.py", line 329, in finished
    self.show_data(justanalyzed=True)
  File "C:\Users\Lizzy\Anaconda3\lib\site-packages\spyder_profiler\widgets\profilergui.py", line 350, in show_data
    self.datatree.load_data(self.DATAPATH)
  File "C:\Users\Lizzy\Anaconda3\lib\site-packages\spyder_profiler\widgets\profilergui.py", line 462, in load_data
    stats_indi = [pstats.Stats(profdatafile),]
  File "C:\Users\Lizzy\Anaconda3\lib\pstats.py", line 72, in __init__
    self.init(arg)
  File "C:\Users\Lizzy\Anaconda3\lib\pstats.py", line 86, in init
    self.load_stats(arg)
  File "C:\Users\Lizzy\Anaconda3\lib\pstats.py", line 99, in load_stats
    with open(arg, 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\Lizzy\\.spyder-py3\\profiler.results'

## Version and main components

* Spyder Version: 3.2.6 
* Python Version: 3.6.4
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.22.0 (OK)
numpy >=1.7      :  1.14.0 (OK)
sphinx >=0.6.6   :  1.6.6 (OK)
rope >=0.9.4     :  0.10.7 (OK)
jedi >=0.9.0     :  0.11.1 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.27.3 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.2.1 (OK)
pylint >=0.25    :  1.8.2 (OK)

```

**Comment by ccordoba12** [comment]
This problem was fixed in our 3.2.7 version. Please update.

**Comment by CAM-Gerlach** [comment]
Fixed as part of #6220 . You can update with ``conda update spyder`` from the Anaconda prompt.
